### PR TITLE
Minor update of bazel doc

### PR DESCRIPTION
**Pull request by harmonyharmo**
Signed-off-by: Yitao Hu <harmonyharmo@users.noreply.github.com>

### CODE REVIEW

File: bazel/README.md [hunk 1]
```diff
@@ -87,6 +87,16 @@ for how to update or override dependencies.
     available via a package manager, but may not be available in default repositories depending on
     OS version. To build against `libc++` build with the `--config=libc++` instead of the
     `--config=clang` flag.
+    
+    #### Config Flag Choices
+    Different config flags specify the compiler libraries: 
+    https://docs.bazel.build/versions/master/guide.html#--config
+    
+    - --config=libc++ => clang + libc++
```
Comment (asraa) [inline bazel/README.md:95]: format nit: wrap --config=libc++ with backquotes like `--config=libc++`

File: bazel/README.md [hunk 2]
```diff
@@ -87,6 +87,16 @@ for how to update or override dependencies.
     available via a package manager, but may not be available in default repositories depending on
     OS version. To build against `libc++` build with the `--config=libc++` instead of the
```
Comment (junr03) [inline bazel/README.md:88]: Let's delete the last sentence of this paragraph, as it is a repetition on the information below.

File: bazel/README.md [hunk 3]
```diff
@@ -87,6 +87,16 @@ for how to update or override dependencies.
...
     `--config=clang` flag.
+    
+    #### Config Flag Choices
+    Different config flags specify the compiler libraries: 
+    https://docs.bazel.build/versions/master/guide.html#--config
```
Comment (junr03) [inline bazel/README.md:93]:
```suggestion
    Different [config](https://docs.bazel.build/versions/master/guide.html#--config) flags specify the compiler libraries:   
```

File: bazel/README.md [hunk 4]
```diff
@@ -87,6 +87,16 @@ for how to update or override dependencies.
...
+    
+    - `--config=libc++` means using `clang` + `libc++`
+    - no config flag means using `gcc` + `libstdc++`
```
Comment (junr03) [inline bazel/README.md:96]: nit: move this last

**Review by junr03 — CHANGES_REQUESTED** [review]
Thanks! A few suggestions

File: bazel/README.md (hunk 2 above)
Comment (harmonyharmo) [reply bazel/README.md:88]: sure

Comment (junr03) [reply bazel/README.md:88]: @harmonyharmo this sentence is still here?

Comment (harmonyharmo) [reply bazel/README.md]: oops, fixed now

**Review by harmonyharmo — COMMENTED** [review]
resubmit changes

**Review by junr03 — APPROVED** [review]

**Pull request merged**